src/pages/product/Product.jsx

```diff
@@ -73,7 +65,7 @@ export const Product = function () {
             <Grid item xs={12} md={5}>
               <ProductDetails {...product} />
               <Divider sx={{ marginTop: "24px", marginBottom: "32px" }} />
-              <ActionsSection onAddToCart={() => addToCart(product.id)} />
+              <ActionsSection onAddToCart={() => Storage.addProduct(product)} productID={product.idء} />
```

Comment:
should be ```productId``` , in camel case only the first letter of every word should be capital